rmw_connext_cpp/src/functions.cpp

```diff
@@ -934,6 +934,47 @@ rmw_create_client(
     goto fail;
   }
 
+  // For now, use the default QoS profile for services.
```

Comment:
Alright, I'll make the necessary changes here and to rmw and rmw_opensplice. I'll leave the work to propagating the QoS profile to rclcpp until after this set gets merged, if that's ok.